utils/api/benefitHandler.ts

```diff
@@ -253,6 +239,27 @@ export class BenefitHandler {
       if (this.input.partner.livedOutsideCanada) {
         requiredFields.push(FieldKey.PARTNER_YEARS_IN_CANADA_SINCE_18)
       }
+
+      /*
+      Make changes to avoid contridictions when showing the Benefit Partner Question: 
+        1. when partner is younger than 60, the partner will not be eligible for OAS, GIS, ALW.
+        2. when partner is between 60 and 65, partner will not be eligible for OAS
+        3. Only show it when: partner is older than 65 AND
+            currently lives in Canada and has lived for 10+ years  OR 
+            currently lives outside Canada and has lived for 20+ years in Canada
+       */
+
+      if (this.input.partner.age > 65) {
```

Comment:
Nested if statements is not great, can try below if you want (need to double check it):
`if (this.input.partner.age > 65 && 
  ((this.input.partner.livingCountry.canada && this.input.partner.yearsInCanadaSince18 > 10) || 
  (!this.input.partner.livingCountry.canada && this.input.partner.yearsInCanadaSince18 > 20))) { 
    requiredFields.push(FieldKey.PARTNER_BENEFIT_STATUS)}`